config.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+# Copyright GPL3.
+# Code by Arthur Milchior <arthur@milchior.fr>
+
+import sys
+
+from aqt import mw
+from aqt.utils import showWarning
+
+userOption = None
+
+
+def _getUserOption():
+    global userOption
+    if userOption is None:
+        userOption = mw.addonManager.getConfig(__name__)
+
+
+def getUserOption(key=None, default=None):
+    """Get the user option if it is set. Otherwise return the default
+    value and add it to the config.
+
+    When an add-on was updated, new config key were not added. This
```

Comment:
Could I ask what you mean by it? Maybe I misunderstood, but I tested with a new addon with 'abc' in its config, then created a new version to also have 'bcd' in the config. I uploaded and downloaded the first version in Anki, then upload-update to the second version. And I could see both 'abc' and 'bcd' in the addon config editor in Anki.